<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FahrplanWidget</class>
 <widget class="QWidget" name="FahrplanWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTabWidget" name="zugliste_widget">
      <widget class="QWidget" name="zugliste_tab">
       <attribute name="title">
        <string>Züge</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="suche_zugnummer_label">
            <property name="text">
             <string>Suche Zugnummer</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="suche_zugnummer_edit"/>
          </item>
          <item>
           <widget class="QLabel" name="suche_gleis_label">
            <property name="text">
             <string>Suche Gleis</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="suche_gleis_edit"/>
          </item>
          <item>
           <widget class="QPushButton" name="suche_loeschen_button">
            <property name="text">
             <string>Suche löschen</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="zugliste_view">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="vorlaufzeit_label">
            <property name="text">
             <string>V&amp;orlaufzeit</string>
            </property>
            <property name="buddy">
             <cstring>vorlaufzeit_spin</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="vorlaufzeit_spin">
            <property name="wrapping">
             <bool>true</bool>
            </property>
            <property name="specialValueText">
             <string>unbegrenzt</string>
            </property>
            <property name="suffix">
             <string> Min.</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>120</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>60</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="nachlaufzeit_label">
            <property name="text">
             <string>N&amp;achlaufzeit</string>
            </property>
            <property name="buddy">
             <cstring>nachlaufzeit_spin</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="nachlaufzeit_spin">
            <property name="wrapping">
             <bool>true</bool>
            </property>
            <property name="specialValueText">
             <string>unbegrenzt</string>
            </property>
            <property name="suffix">
             <string> Min.</string>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>120</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>60</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="fahrplan_tab">
       <attribute name="title">
        <string>Fahrplan</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QSplitter" name="fahrplan_splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QLabel" name="fahrplan_label">
           <property name="text">
            <string>Fahrplan</string>
           </property>
           <property name="buddy">
            <cstring>fahrplan_view</cstring>
           </property>
          </widget>
          <widget class="QTableView" name="fahrplan_view">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="folgezug_label">
           <property name="text">
            <string>Folgezug</string>
           </property>
           <property name="buddy">
            <cstring>folgezug_view</cstring>
           </property>
          </widget>
          <widget class="QTableView" name="folgezug_view">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="grafik_tab">
       <attribute name="title">
        <string>Grafik</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QScrollArea" name="grafik_area">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="grafik_widget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>515</width>
             <height>736</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
